src/naf-dialog-adapter.js

```diff
@@ -146,6 +150,24 @@ export default class DialogAdapter {
     this._onOccupantMessage = messageListener;
   }
 
+  // Add an event listener for a peer and an event type
+  addEventListener(peerId, eventType, listener) {
```

Comment:
No reason at all, just what I came up with at the time. Regarding `EventEmitter` I saw it but I couldn't see it being used anywhere else but I'm happy to change to a well known and tested solution. For the scene events I considered it but just prefer each class to be responsible for it's own events rather than throwing everything inside the scene unless it's a global event that's going to be emitted across the board.